x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/settings/components/fleet_server_hosts_flyout/ssl_form_section.tsx

```diff
@@ -93,13 +93,14 @@ export const SSLFormSection: React.FunctionComponent<Props> = (props) => {
             placeholder={i18n.translate(
               'xpack.fleet.settings.fleetServerHosts.sslCertificateAuthoritiesInputPlaceholder',
               {
-                defaultMessage: 'Specify certificate authority',
+                defaultMessage:
+                  'Specify the CA certificate(s) that signed the Elastic Agent client certificates connecting to this Fleet Server. Fleet Server trusts these CAs when verifying incoming connections.',
               }
             )}
             label={i18n.translate(
               'xpack.fleet.settings.fleetServerHosts.sslCertificateAuthoritiesInputLabel',
               {
-                defaultMessage: 'Server SSL certificate authorities',
+                defaultMessage: 'Fleet Server > Elastic Agents certificate authorities',
```

Comment:
the renames are the same as https://github.com/elastic/kibana/pull/236887, these changes will go away when the earlier pr is merged